### ISSUE TITLE
Allow Dropzone to take file name for PreviewList

### ISSUE THREAD

**Issue by simranbling** [Yuvaleros/material-ui-dropzone#44]
I have the dropzone in a stepper. If I loose the focus, I want to re-display the files that were initially uploaded.

**Comment by Yuvaleros** [comment]
Sorry for the late response.
I don't really understand the scenario, could you rephrase? maybe add an image?

**Comment by panz3r** [comment]
This issue should be resolved by the `initialFiles` prop added to `DropzoneArea` component.
